src/styles/index.scss

```diff
@@ -1,3 +1,69 @@
+@mixin center {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+@keyframes rotate-block {
+  from {
+    transform: translate(-50%, -100%) rotate(0deg);
+  }
+  to {
+    transform: translate(-50%, -100%) rotate(360deg);
+  }
+}
+
 body {
   margin: 0;
 }
+
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  @include center;
+
+  &__minutes {
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+
+    transform-origin: 50% 100%;
```

Comment:
The `transform-origin` property is duplicated in both `.stopwatch__minutes` and `.stopwatch__seconds`. To follow DRY principles (Don't Repeat Yourself), consider creating a mixin for shared styles between these elements.